mithril-signer/tests/test_extensions/state_machine_tester.rs

```diff
@@ -309,26 +309,51 @@ impl StateMachineTester {
     }
 
     /// cycle the state machine and test the resulting state
-    pub async fn cycle_registered(&mut self) -> Result<&mut Self> {
+    pub async fn cycle_ready_to_sign(&mut self) -> Result<&mut Self> {
         self.cycle().await?;
 
         self.assert(
-            self.state_machine.get_state().await.is_registered(),
+            self.state_machine.get_state().await.is_ready_to_sign(),
             format!(
-                "state machine is in {} state (Registered was expected)",
+                "state machine is in {} state (ReadyToSign was expected)",
                 self.state_machine.get_state().await
             ),
         )
     }
 
+    pub async fn cycle_ready_to_sign_no_registration(&mut self) -> Result<&mut Self> {
```

Comment:
The name is difficult to understand, maybe this one is better:
```suggestion
    pub async fn cycle_ready_to_sign_without_signature_registration(&mut self) -> Result<&mut Self> {
```